lib/graphql/schema/field.rb

```diff
@@ -102,16 +102,16 @@ def initialize(type: nil, name: nil, owner: nil, null: nil, field: nil, function
         if name.nil?
           raise ArgumentError, "missing first `name` argument or keyword `name:`"
         end
-        if !(field || function || mutation || resolver)
+        if !(field || function || resolver_class)
```

Comment:
`mutation` and `resolver` always return nil because `@resolver_class` is not yet initialized here. I think we can use the `resolver_class` argument instead of these methods.